example_configs/pinot.yml

```diff
@@ -0,0 +1,208 @@
+rules:
+# Pinot Controller
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"ControllerMetrics\", name=\"pinot.controllercontroller(\\w+)\"><>(\\w+)"
+  name: "pinot_controller_$1_$2"
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"ControllerMetrics\", name=\"pinot.controllerhelix\\.(\\w+)\"><>(\\w+)"
+  name: "pinot_controller_helix_$1_$2"
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"ControllerMetrics\", name=\"pinot.controllerhelixZookeeperReconnects\"><>(\\w+)"
+  name: "pinot_controller_helix_ZookeeperReconnects_$1"
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"ControllerMetrics\", name=\"pinot.controlleridealstateZnodeSize.(\\w+)_(\\w+)\"><>(\\w+)"
+  name: "pinot_controller_idealstateZnodeSize_$3"
+  labels:
+    table: "$1"
+    tableType: "$2"
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"ControllerMetrics\", name=\"pinot.controllernumberOfReplicas.(\\w+)_(\\w+)\"><>(\\w+)"
```

Comment:
I think you can combine many of these rules, as they're following the same pattern.